runtime/bcverify/staticverify.c

```diff
@@ -1421,19 +1429,19 @@ checkStackMapEntries (J9CfrClassFile* classfile, J9CfrAttributeCode * code, U_8
 			}
 		}
 
-		/* locals:
-		 * 		It is an error if, for any index i, locals[i] represents a local variable whose index is greater than the maximum 
-		 * 		number of local variables for the method. 
-		 * stack:
-		 * 		It is an error if, for any index i, stack[i] represents a stack entry whose index is greater than the maximum operand stack size for the method.
-		 */
-		slotTypeCounter += 1;
-		if ((CFR_STACKMAP_TYPE_DOUBLE == entryType) || (CFR_STACKMAP_TYPE_LONG == entryType)) {
+		/* For StackMapTable entries with local variable arrays a value of type long or double must occupy two consecutive 
+		 * local variables. Ensure that if there is a long or double entry the array reflects the correct number of 
+		 * slots. An incorrect maxLocals value in all other cases will be handled in bcverify.c to match the RI. Don't bother
+		 * checking the final slot as this is considered out of bounds and will be handled later. */
```

Comment:
>  will be handled later.

Any issue with letting it be handled here?  Do the relevant tests still pass and generate the same exception?